<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodecCompareWindow</class>
 <widget class="QWidget" name="CodecCompareWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,10,3,0">
       <item row="1" column="0">
        <widget class="VlcWidgetVideo" name="rawVideo" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="original">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_1" native="true">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>52</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>zzzzzzzz</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="frames0">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,3">
       <item>
        <widget class="QLabel" name="first">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo1" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,3">
       <item>
        <widget class="QLabel" name="second">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo2" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
         <zorder>encodedVideo1</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10,3">
       <item>
        <widget class="QLabel" name="third">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo3" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
         <zorder>rawVideo</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <widget class="QTextEdit" name="frames3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>50</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>10</y>
            <width>52</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>hmm</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
